point-of-sale/src/pages/new.tsx

```diff
@@ -1 +1,12 @@
 export { default } from '../client/components/pages/NewPage';
+
+export function getServerSideProps() {
+  console.log('Running PendingPage getServerSideProps');
```

Comment:
nit: `PendingPage` -> `NewPage`

Might not need the console log at all though.